tf2onnx/optimizer/merge_duplicated_nodes_optimizer.py

```diff
@@ -90,3 +92,41 @@ def _skip_node_type(node):
     @staticmethod
     def _len_of_node_output(node):
         return len(node.output)
+
+    @staticmethod
+    def _are_attr_equal(a, b):
+        if len(a) != len(b):
+            return False
+        for k, v in a.items():
+            if v.type == onnx_pb.AttributeProto.FLOAT or \
+                    v.type == onnx_pb.AttributeProto.INT or \
+                    v.type == onnx_pb.AttributeProto.STRING or \
+                    v.type == onnx_pb.AttributeProto.FLOATS or \
+                    v.type == onnx_pb.AttributeProto.INTS or \
+                    v.type == onnx_pb.AttributeProto.STRINGS:
+                if v != b[k]:
```

Comment:
the onnx_pb.AttributeProto.FLOATS doesn't have  "raw_data" field?

Reply:
I think the plain old `==` works fine for all types except tensor and graph

Reply:
according to my understanding, **the plain old == is too much strict** so the optimizer don't merge node that can be merged actually even they have some different attributes, am i right?

Reply:
I don't know. What I'm saying is that we have been using `==` with no issues other than the one I described here so we should be able to continue using it when we don't have a better way of comparing them